asserts.cabal

```diff
@@ -0,0 +1,55 @@
+--
```

Comment:
I wonder if it'd make sense to write a comment about why we have this file.

Reply:
Will do :+1: